cypress/support/pages/yves/multi-factor-auth/repositories/b2b-mp-multi-factor-auth-repository.ts

```diff
@@ -0,0 +1,51 @@
+import { injectable } from 'inversify';
+import { MultiFactorAuthRepository } from '../multi-factor-auth-repository';
+
+@injectable()
+export class B2bMpMultiFactorAuthRepository implements MultiFactorAuthRepository {
+  private readonly SELECTORS = {
+    VERIFICATION_CODE_INPUT: 'input[name="codeValidationForm[authentication_code]"]',
+    VERIFY_BUTTON: 'form[name="codeValidationForm"] button[type="submit"]',
+    VERIFICATION_POPUP: '.js-multi-factor-authentication-handler__popup-content',
+    FORM_BOX: '.form.box',
```

Comment:
We can't rely on these classes in locators. We can use:
- [data-qa] attributes
- ID
- name
- href

If current FE implementation doesn't have any of those, it needs to be added
Convention: https://spryker.atlassian.net/wiki/spaces/CCS/pages/4034854963/What+DOM+selectors+to+use+in+Cypress

Reply:
still valid, we can't rely on classes in locators
```
getVerificationPopup(): Cypress.Chainable {
    return cy.get('.js-multi-factor-authentication-handler__popup-content');
  }
```